frontend/containers/DocumentEdit/DocumentEdit.jsx

```diff
@@ -224,7 +226,9 @@ class DocumentEdit extends Component {
   }
 
   componentWillUnmount() {
+    const {actions} = this.props
```

Comment:
Could you add [a space after initialisation](https://github.com/dadi/publish/blob/master/docs/coding-guidelines.md#white-space)?